priv/perf/apps/load_testing/lib/load_test/watcher_info.ex

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule LoadTesting.LoadTest.WatcherInfo do
```

Comment:
hmm, why not name the app `load_test`? I haven't seen verbs used as application names

Reply:
It is a gerund, not a verb. I do not like `load_test` as an app name, but it is just me. I can change it if you want 😉